lib/Cake/I18n/I18n.php

```diff
@@ -727,14 +727,20 @@ protected function _parseLiteralValue($string) {
  *
  * @param string $format Format to be translated
  * @param string $domain Domain where format is stored
+ * @param string $context Context The context of the translation, e.g a verb or a noun.
  * @return mixed translated format string if only value or array of translated strings for corresponding format.
  */
-	protected function _translateTime($format, $domain) {
-		if (!empty($this->_domains[$domain][$this->_lang]['LC_TIME'][$format])) {
-			if (($trans = $this->_domains[$domain][$this->_lang][$this->category][$format])) {
-				return $trans;
-			}
+	protected function _translateTime($format, $domain, $context) {
+		$lang = $this->_lang;
+		$category = $this->category;
+		$trans = Hash::get($this->_domains, "$domain.$lang.$category.$format.$context");
```

Comment:
I suppose for lc_time context will always be an empty string, but it is needed to fetch the translation for lc_time po files from the correct index in the translations array ($_domains).